### PR TITLE
Update accessibility page

### PR DESCRIPTION
**Pull request by nickbristow**
Updating the accessibility page as discussed

### CODE REVIEW

**Comment by gbinal** [comment]
Looks good to me.  Let's add.

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ logs
 
 # virtualenv
 .Python
+.python-version
```
Comment (konklone) [inline .gitignore]: Were you having issues with this, @nickbristow?

**Comment by konklone** [comment]
:+1:, but a note here that we should remember to remove the `static/data/` file this PR added later. That directory was removed in prior PRs during the refactor.

File: .gitignore (hunk 1 above)
Comment (nickbristow) [inline .gitignore]: @konklone I installed pyenv and it added that file, I assumed we didn't need that on github so I added to .gitignore

**Pull request merged**